# A whole set of predicate assignments
#
####

predicate_msgs/PredicateAssignment[] assignments
